src/features/rewards/modalBackupRestore/index.tsx

```diff
@@ -77,145 +84,200 @@ export default class ModalBackupRestore extends React.PureComponent<Props, State
 
   setRecoveryKey = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
     this.setState({
+      errorShown: false,
       recoveryKey: event.target.value
     })
   }
 
   onRestore = (key?: string) => {
     key = typeof key === 'string' ? key : this.state.recoveryKey
-    this.setState({
```

Comment:
Per design, we want to retain the recovery key in case of a problem

cc: @jenn-rhim